happs/proposal/app.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "HoloREA: proposal",
+  "description": "Proposal DNA for the REA economic network",
```

Comment:
We need a better app store description from the core VF docs somewhere. @fosterlynn any suggestions?

Reply:
Hmmm. How about something like

"Publication of intents (requests or offers), often with what is expected in return."

Reply:
I'll add it